package wire.major;   // -*-c-*-

enum Major {
  HELLO_CLIENT_GREETING = 1;
  HELLO_SERVER_GREETING = 2;
  TERRAIN_VIEW_CHANGE = 3;
  TERRAIN = 4;
  TERRAIN_EDIT = 5;
  ENTITY_BECOME_PLAYER = 10;
  ENTITY_PLAYER_STATUS = 11;
  ENTITY_ENTITY_INFO = 12;
  ENTITY_ENTITY_TYPE = 13;
  ENTITY_TELL = 14;
  ENTITY_SCRIPT = 15;
};
